site/app/models/gradeable/GradedVersion.php

```diff
@@ -0,0 +1,186 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: mackek4
+ * Date: 6/19/2018
+ * Time: 8:55 AM
+ */
+
+namespace app\models\gradeable;
+
+
+use app\libraries\Core;
+use app\libraries\DateUtils;
+use app\models\AbstractModel;
+
+/**
+ * Class GradedVersion
+ * @package app\models\gradeable
+ *
+ * Data about and results of autograding for one submission version
+ *
+ * @method int getVersion()
+ * @method float getNonHiddenHonExtraCredit()
+ * @method float getNonHiddenExtraCredit()
+ * @method float getHiddenNonExtraCredit()
+ * @method float getHiddenExtraCredit()
+ * @method \DateTime getSubmissionTime()
+ * @method isAutogradingComplete()
+ */
+class GradedVersion extends AbstractModel {
```

Comment:
GradedGradeableVersion? I know the original class was called GradeableVersion

Reply:
Actually it seems like this class is mostly autograding results. Maybe it should be named something that indicates that.

Reply:
I renamed it to `AutogradingVersion` since it does only contain information about autograding